.github/workflows/helm.yaml

```diff
@@ -43,8 +43,6 @@ jobs:
         with:
           # Fetch entire history. Required for chart-releaser; see https://github.com/helm/chart-releaser-action/issues/13#issuecomment-602063896
           fetch-depth: 0
-      - name: fetch head of branch to ensure semantic can run
```

Comment:
Is this step also not needed

Reply:
hmm, this should not have been part of this change, will revert